### DISCUSSION TITLE
Ugly Drum Smoker / Charcoal Airflow only

### DISCUSSION THREAD

**Discussion by volkswatson** [nebhead/PiFire#139] [Ideas]
Hey there, I am an Ugly Drum Smoker guy, and I'd love to implement this on it. (it's basically a drum smoker with charcoal basket).

I've seen suggestions of HeaterMeter, but it looks like their site has been offline for a while (not to mention a little pricer than I'd like)

Are there any plans to add a PWM control for intake control? Would prob need some sort of airflow shutoff control as well.

Anyway, I love the PiFire interface and would really love to use it for my application if possible. Thanks!

**Comment by nebhead** [comment]
Hi @volkswatson - No current plans for a PWM controller for air intake controls at the moment, but it does sound like an interesting fork that someone could take for this project.  I lifted some of the HW design ideas (i.e. ADC for temps) from HeaterMeter in the first place, so some of it's not too far off already.  Not sure if anyone out there in the community would be interested, but I think it would be a cool addition to the project.

**Comment by volkswatson** [comment]
Yeah, there’s a pretty good community for UDS. I’d love to see this implemented for a simple smoker. I actually scoured the Internet for a while to look for some options but honestly, this is a better solution than anything I found.On Nov 7, 2022, at 5:59 PM, nebhead ***@***.***> wrote:﻿
Hi @volkswatson - No current plans for a PWM controller for air intake controls at the moment, but it does sound like an interesting fork that someone could take for this project.  I lifted some of the HW design ideas (i.e. ADC for temps) from HeaterMeter in the first place, so some of it's not too far off already.  Not sure if anyone out there in the community would be interested, but I think it would be a cool addition to the project.

**Reply by nebhead** [reply]
I just checked out a YouTube video for these and they are cool!  I might have to start looking into this to see how I can adapt it.
